src/App.tsx

```diff
@@ -1,61 +1,135 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  // if there is no user with a given userId
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [currentTodos, setCurrentTodos] = useState<Todo[]>(todos);
+
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
+
+  const [option, setOption] = useState(0);
+  const [userError, setUserError] = useState(false);
+
+  const pushUser = (todo: Todo) => {
+    const newTodo = {
+      ...todo,
+    };
```

Comment:
You don't need to create a new object. `todo` has just been created in handleSubmit